bigquery_etl/cli/query.py

```diff
@@ -954,6 +956,11 @@ def _run_query(
 
             query_arguments.append("--destination_table={}".format(destination_table))
 
+        if mapped_values is not None and mapped_values != "":
+            query_arguments = query_arguments + [
+                f"--parameter=sample_id:INT64:{mapped_values}"
+            ]
```

Comment:
I hate to do this to you, but `query_arguments` is passed to this function. We can do this in the calling function and leave `_run_query` as-is

Reply:
Interesting, how would that be?
The [mapped_values is the iterable](https://github.com/mozilla/bigquery-etl/blob/50f99cf47b4a390af7cc32d0b721626803b52bcc/bigquery_etl/cli/query.py#L1276), so not known in advance in the calling function. Even calling it as
```
pool.map(
  partial
  (
      _run_query,
      arguments + [f"--parameter=sample_id:INT64:{mapped_values}"] if mapped_values != "" else arguments,```
  )
```
would result in the complete list passed.

Reply:
Something like this:
```
pool.map(
            partial(
                _run_query,
                [query_file],
                project,
                None,
                table,
                dataset,
                addl_templates,
            ),
            [f"--parameter=sample_id:INT64:{v}" for v in mapped_values],
        )
```

Reply:
- The `query_arguments` argument comes before the `addl_templates` argument, so I think `addl_templates` needs to be specified as a keyword argument for this to work.
- The `query_arguments` argument should be a list.
```python
        pool.map(
            partial(
                _run_query,
                [query_file],
                project,
                None,
                table,
                dataset,
                addl_templates=addl_templates,
            ),
            [[f"--parameter=sample_id:INT64:{v}"] for v in mapped_values],
        )
```